Testing/SystemTests/tests/framework/CalibrateRectangularDetector_Test.py

```diff
@@ -26,6 +26,10 @@ def cleanup(self):
         os.remove(self.saved_cal_file)
 
     def skipTests(self):
+        # don't run in conda, temporary solution
+        if "CONDA_PREFIX" in os.environ:
+            return True
```

Comment:
This is the sort of thing which gets lost when we switch to Conda proper and nobody every comes back to fix it, do we know why we cannot run this test?

Reply:
An issue was made that should address this, it's because we are testing a dependency as it is a none-unique solution to a problem.